main/tests/test_main.cpp

```diff
@@ -34,7 +34,11 @@
 #ifdef DEBUG_ENABLED
 
 #include "test_astar.h"
+
+#ifdef GDSCRIPT_ENABLED
```

Comment:
It would be better to move it after the main block of includes, so that it doesn't introduce an unnecessary split between `test_astar.h` and the rest.

Reply:
Done.